packages/rest/src/__tests__/acceptance/coercion/coercion.acceptance.ts

```diff
@@ -94,13 +111,10 @@ describe('Coercion', () => {
       })
       .expect(200);
     sinon.assert.calledWithExactly(spy, {
-      // Notice that numeric and boolean values are converted to strings.
-      // This is because all values are encoded as strings on URL queries
-      // and we did not specify any schema in @param.query.object() decorator.
```

Comment:
Please preserve the "notice" comment and update it to describe (point out) the new behavior.

Reply:
On the second though, I think we should preserve the current test as-is to describe how are schema-less values handled, and add a new test to verify how does schema-based coercion work.

Reply:
> I think we should preserve the current test as-is to describe how are schema-less values handled, and add a new test to verify how does schema-based coercion work.

Good point, I will edit the test.